x-pack/plugins/stack_connectors/common/experimental_features.ts

```diff
@@ -15,6 +15,7 @@ export const allowedExperimentalValues = Object.freeze({
   isMustacheAutocompleteOn: false,
   sentinelOneConnectorOn: true,
   crowdstrikeConnectorOn: true,
+  inferenceConnectorOn: true,
```

Comment:
Is the intention here to be able to turn the feature off? Feature flags can only be set from `false` to `true`. As you can see in these screenshots of dev.yml, browser logs, and server logs, whether or not the feature flag is included in dev.yml the value is always `true`:

#### Flag is in dev yml
<img width="644" alt="Screenshot 2024-10-09 at 9 37 11 AM" src="https://github.com/user-attachments/assets/440832d8-1a64-4feb-8f81-b5c97f01f369">

#### Flag is NOT in dev yml
<img width="452" alt="Screenshot 2024-10-09 at 9 37 51 AM" src="https://github.com/user-attachments/assets/0614f459-efa2-421d-b1f6-1b0857059e22">


If you want to be able to turn the flag off from dev yml, you should name it `inferenceConnectorOff` and set it to `false`. Reverse your conditions in the code. Then when `inferenceConnectorOff` is added to dev yml, the value will change from `false` to `true` and you can remove your feature.
```suggestion
  inferenceConnectorOff: false,
```

Reply:
Yes, it's on purpose to be able to show Technical preview badge. But it should be available from the beginning. Similar is happening for Crowdstrike connector